packages/docs/pages/apps/google-docs/actions.md

```diff
@@ -0,0 +1,12 @@
+---
+favicon: /favicons/google-docs.svg
+items:
+  - name: Create new document
+    desc: Create a blank Google Doc which takes 2 parameters - title and content.
```

Comment:
Not necessarily "blank" here, as it optionally receives content for the file.


```suggestion
  - name: Create document
    desc: Create a blank document with a title. Optionally, provide content.
```
should suffice here.